packages/components/src/DateTimePickers/InputDateTimePicker/__snapshots__/InputDateTimePicker.component.test.js.snap

```diff
@@ -207,7 +207,7 @@ exports[`InputDateTimePicker should render 1`] = `
                     <span
                       style="font-weight: 400; font-size: 1.6rem; visibility: hidden; position: absolute;"
                     >
-                      15:27:00
```

Comment:
this snapshot change is because now we only use placeholder text to measure the width of input.